exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/OperatorWrapper.java

```diff
@@ -179,12 +207,47 @@ public String getMetricsTable() {
       }
       for (final Number value : values) {
         if (value != null) {
-          builder.appendFormattedNumber(value, null);
+          builder.appendFormattedNumber(value);
         } else {
-          builder.appendCell("", null);
+          builder.appendCell("");
         }
       }
     }
     return builder.build();
   }
+
+  private class OverviewTblTxt {
+    static final String OperatorID = "Operator ID";
+    static final String Type = "Type";
+    static final String AvgSetupTime = "Avg Setup Time";
+    static final String MaxSetupTime = "Max Setup Time";
+    static final String AvgProcessTime = "Avg Process Time";
+    static final String MaxProcessTime = "Max Process Time";
+    static final String MinWaitTime = "Min Wait Time";
+    static final String AvgWaitTime = "Avg Wait Time";
+    static final String MaxWaitTime = "Max Wait Time";
+    static final String PercentFragmentTime = "% Fragment Time";
+    static final String PercentQueryTime = "% Query Time";
+    static final String Rows = "Rows";
+    static final String AvgPeakMemory = "Avg Peak Memory";
+    static final String MaxPeakMemory = "Max Peak Memory";
+  }
+
+  private class OverviewTblTooltip {
+    static final String OperatorID = "Operator ID";
+    static final String Type = "Operator Type";
+    static final String AvgSetupTime = "Average Time in setting up fragments";
+    static final String MaxSetupTime = "Longest Time a fragment took in setup";
+    static final String AvgProcessTime = "Average processing time for a fragment";
+    static final String MaxProcessTime = "Longest Time a fragment took to process";
+    static final String MinWaitTime = "Shortest time a fragment spent in waiting for data";
+    static final String AvgWaitTime = "Average wait time for a fragment";
+    static final String MaxWaitTime = "Longest Time a fragment spent in waiting data";
+    static final String PercentFragmentTime = "Percentage of the total fragment time that was spent on the operator";
+    static final String PercentQueryTime = "Percentage of the total query time that was spent on the operator";
+    static final String Rows = "Rows emitted by the operator";
```

Comment:
Rows emitted by scans is incomplete. Operators like Filter, HashAgg, etc will also change the number of outgoing rows wrt the number of incoming rows. 
I think we should leave this as is 'Rows emitter by the operator'